### PR TITLE
add endpoint missed in last pr for bulk point of interest fetch

### PR DESCRIPTION
**Pull request by micheal-w-wells**

### CODE REVIEW

File: api-mobile/src/paths/points-of-interest.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { SQLStatement } from 'sql-template-strings';
+import { ALL_ROLES } from '../constants/misc';
+import { getDBConnection } from '../database/db';
+import { PointOfInterestSearchCriteria } from '../models/point-of-interest';
+import geoJSON_Feature_Schema from '../openapi/geojson-feature-doc.json';
+import { getPointOfInterestSQL, getPointsOfInterestSQL } from '../queries/point-of-interest-queries';
```
Comment (NickPhura) [inline api-mobile/src/paths/points-of-interest.ts:10]: nit: the `getPointOfInterestSQL` import isnt used

**Review by NickPhura — APPROVED** [review]

**Comment by NickPhura** [comment]
@micheal-w-wells Can this be merged, or are you adding more to it?

**Comment by micheal-w-wells** [comment]
@NickPhura i totally thought it already was, mystery solved.

**Comment by micheal-w-wells** [comment]
i'll grab the unused import on next pr

**Pull request merged**